src/Serilog.Sinks.Elasticsearch/Sinks/ElasticSearch/ElasticsearchSinkOptions.cs

```diff
@@ -76,6 +76,14 @@ public class ElasticsearchSinkOptions
         /// </summary>
         public int? NumberOfReplicas { get; set; }
 
+        /// <summary>
+        /// Index aliases. Used in rollover jobs and etc.
```

Comment:
Can you elaborate what happens when you set the value?

Reply:
I've changed comment. Index alias is the level property as NumberOfReplicas. Hope this comment is enough :).